pkg/ebpf/c/bpf_cross_compile.h

```diff
@@ -161,7 +161,7 @@
 #define __E_BPF_FUNC_set_hash_invalid true
 #endif
 
-#endif /* LINUX_VERSION_CODE < KERNEL_VERSION(4, 10, 0) */
+#endif /* LINUX_VERSION_CODE < KERNEL_VERSION(4, 12, 0) */
```

Comment:
I think this was 4.10 because that was when `__BPF_FUNC_MAPPER` was added.

Reply:
Ah this was a leftover from a previous attempt to fix the errors. Removing it